src/Toolkit/Toolkit.Maui/BookmarksView/BookmarksView.cs

```diff
@@ -35,18 +35,17 @@ static BookmarksView()
     {
         DefaultDataTemplate = new DataTemplate(() =>
         {
-            var defaultCell = new TextCell();
-            defaultCell.SetBinding(TextCell.TextProperty, nameof(Bookmark.Name));
-            return defaultCell;
+            var defaultLabel = new Label();
+            defaultLabel.SetBinding(Label.TextProperty, nameof(Bookmark.Name));
+            return defaultLabel;
         });
 
-        string template = @"<ControlTemplate xmlns=""http://schemas.microsoft.com/dotnet/2021/maui"" xmlns:x=""http://schemas.microsoft.com/winfx/2009/xaml"" xmlns:esriTK=""clr-namespace:Esri.ArcGISRuntime.Toolkit.Maui"">
-                                    <ListView x:Name=""PresentingView"" HorizontalOptions=""FillAndExpand"" VerticalOptions=""FillAndExpand"">
-                                        <x:Arguments>
-                                            <ListViewCachingStrategy>RecycleElement</ListViewCachingStrategy>
-                                        </x:Arguments>
-                                    </ListView>
-                                </ControlTemplate>";
+        string template =
+            $@"<ControlTemplate xmlns=""http://schemas.microsoft.com/dotnet/2021/maui""
+                               xmlns:x=""http://schemas.microsoft.com/winfx/2009/xaml""
+                               xmlns:esriTK=""clr-namespace:Esri.ArcGISRuntime.Toolkit.Maui"">
+                <CollectionView x:Name=""{_presentingViewName}"" SelectionMode=""Single"" />
```

Comment:
You should add `ItemTemplate="{TemplateBinding ItemTemplate}"` here, instead of having to deal with it in the code.